frontend/cypress/fixtures/mixedcase-account-ids.json

```diff
@@ -1 +1 @@
-["near", "neAR", "FOUNdation.NEAR"]
+["something.near", "sOmEtHiNg.near", "something.NEAR"]
```

Comment:
I changed those because we wait too long for foundation on SSR.
Actually, I fixed the wait, but decided to keep simpler accounts anyway.